canonical/ubuntu-pro-client
# Add vulnerabilities API

## Why is this needed?
We are adding support for the `u.pro.security.vulnerabilities.cve.v1` endpoint.
This endpoint will list all the fixable CVEs in the system.

This is a draft PR just for us to align on the parsing strategy here.
I still need to write all of the tests for this feature (unit and integration)

## Test Steps
1. Launch a xenial container
2. Run `pro api u.pro.security.vulnerabilities.cve.v1`
3. See that we have valid CVEs listed in the output
4. Attach the machine to a Pro subscription
5. Run `apt upgrade`
6. Run the CLI command again and verify that no CVEs are listed now

PS: I still need to write the unit and integration tests for this feature


---
